lms/djangoapps/bookmarks/tests/test_views.py

```diff
@@ -139,8 +139,8 @@ def send_delete(self, client, url, expected_status=204):
 class BookmarksListViewTests(BookmarksViewTestsMixin):
     """
     This contains the tests for GET & POST methods of bookmark.views.BookmarksListView class
-    GET /api/bookmarks/v0/bookmarks/?course_id={course_id1}
-    POST /api/bookmarks/v0/bookmarks
+    GET /api/bookmarks/v1/bookmarks/?course_id={course_id1}
```

Comment:
We decided to turn API to v1 in this PR instead of changing in UI. So that we can have whole feature in working condition.